appium/webdriver/common/multi_action.py

```diff
@@ -68,7 +76,7 @@ def perform(self):
         return self
 
     @property
-    def json_wire_gestures(self):
+    def json_wire_gestures(self) -> Dict:
```

Comment:
Dict could also be parametrised with key/value types, for example Dict[str, str]